src/plugins/help/help.js

```diff
@@ -21,46 +16,88 @@
     var rows = [];
     var timeoutHandle;
 
-    var triggerEvent = function( el, eventName, detail ) {
-        var event = document.createEvent( "CustomEvent" );
-        event.initCustomEvent( eventName, true, true, detail );
-        el.dispatchEvent( event );
+    var body = document.querySelector( "body" );
+
+    // This querySelector is used to keep compatibility with templates using div#impress-help
+    var helpDiv = document.querySelector( "#impress-help" ) || document.createElement( "div" );
+    body.appendChild( helpDiv );
+
+    var fullHelpDiv = document.createElement( "div" );
+    var fullHelpStyle = {
+        display: "none",
+        position: "fixed",
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
+        backgroundColor: "#000",
+        color: "#FFF",
+        opacity: 0.8,
+        textAlign: "center",
+        padding: "3em"
```

Comment:
The philosophy of impress.js is that we don't want to force any colors or other styling for the user. So instead of setting the style here, you should set it in css/impress-demo.css. Unfortunately this means you also need to set this in every other css file under presentations. (Related: #681)

So what is left for the plugin to do? Maybe nothing? Do you need different CSS for the full help?

At most, you could set a class that indicates short help vs full help:

    <div id="impress-help" class="help-short">
    <div id="impress-help" class="help-full">

Presentation authors can then choose to use this class for styling if they want.